packages/editor-ui/src/components/Node.vue

```diff
@@ -256,6 +355,16 @@ export default mixins(externalHooks, nodeBase, nodeHelpers, workflowHelpers).ext
 				this.$emit('duplicateNode', this.data.name);
 			});
 		},
+		getTriggerNodeTooltip (): string | undefined {
+			if (this.nodeType !== null && this.nodeType.hasOwnProperty('eventTriggerDescription')) {
+				return this.nodeType.eventTriggerDescription;
+			} else {
+				return `Waiting for you to create an event in ${this.getTrimmedWebhookNodeName()}`;
+			}
+		},
+		getTrimmedWebhookNodeName() {
```

Comment:
rename to `getTrimmedTriggerNodeTypeName` which is more accurate

Reply:
Renamed.